addons/l10n_it/models/account_move.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    def _extend_with_attachments(self, attachments, new=False):
+
```

Comment:
I see no added value in having an empty line here; although I usually enjoy putting empty lines to better structure the code and make it more readable.